### DISCUSSION TITLE
Third Party Agents installation on Strongswann Ubuntu VPN

### DISCUSSION THREAD

**Discussion by thatsdanish** [strongswan/strongswan#359] [Q&A]
I have a AWS Strongswann VPN running on UBUNTU 16.04..for compliance there is a requirement to install Datadog, Antivirus , VA Agents & Patches. i was wondering if i can install these agents on the ubuntu strongswann server or a waiver will be required.

Can these thirdparty agents installed on the server without disrupting server functionality ? 

kindly help

**Comment by Thermi** [comment]
It is unlikely to cause issues.

Am April 28, 2021 12:57:19 PM UTC schrieb thatsdanish ***@***.***>:
>I have a AWS Strongswann VPN running on UBUNTU 16.04..for compliance
>there is a requirement to install Datadog, Antivirus , VA Agents &
>Patches. i was wondering if i can install these agents on the ubuntu
>strongswann server or a waiver will be required.
>
>Can these thirdparty agents installed on the server without disrupting
>server functionality ? 
>
>kindly help
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/strongswan/strongswan/discussions/359

-- 
Sent from mobile
